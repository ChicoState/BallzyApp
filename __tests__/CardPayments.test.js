'use strict';

import 'react-native';
import React from 'react';
//import React from 'react';
import CardPay from '../src/components/payments/CardPayments';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';

test('renders correctly', () => {
  const tree = renderer.create(
    <CardPay />
  ).toJSON();
  expect(tree).toMatchSnapshot();

});
