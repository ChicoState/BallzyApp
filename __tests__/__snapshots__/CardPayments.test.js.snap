exports[`test renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "stretch",
      "backgroundColor": "#00BFFF",
      "flex": 1,
    }
  }>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "fontSize": 40,
        "fontWeight": "bold",
        "marginBottom": 75,
        "marginTop": 75,
        "textAlign": "center",
      }
    }>
    Select Your Card
  </Text>
  <View
    style={
      Object {
        "flex": 20,
        "flexDirection": "column",
      }
    }>
    <View
      style={
        Object {
          "flex": 1,
          "flexDirection": "row",
        }
      }>
      <View
        accessibilityComponentType={undefined}
        accessibilityLabel={undefined}
        accessibilityTraits={undefined}
        accessible={true}
        hitSlop={undefined}
        isTVSelectable={true}
        onLayout={undefined}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "opacity": 1,
          }
        }
        testID={undefined}
        tvParallaxProperties={undefined}>
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Object {
              "backgroundColor": "#42033D",
              "borderColor": "black",
              "borderWidth": 2,
              "color": "#F7FFF7",
              "fontSize": 16,
              "height": 100,
              "margin": 20,
              "width": 160,
            }
          }>
          Amex
        </Text>
      </View>
      <View
        accessibilityComponentType={undefined}
        accessibilityLabel={undefined}
        accessibilityTraits={undefined}
        accessible={true}
        hitSlop={undefined}
        isTVSelectable={true}
        onLayout={undefined}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "opacity": 1,
          }
        }
        testID={undefined}
        tvParallaxProperties={undefined}>
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Object {
              "backgroundColor": "#42033D",
              "borderColor": "black",
              "borderWidth": 2,
              "color": "#F7FFF7",
              "fontSize": 16,
              "height": 100,
              "margin": 20,
              "width": 160,
            }
          }>
          Visa
        </Text>
      </View>
    </View>
    <View
      style={
        Object {
          "flex": 1,
          "flexDirection": "row",
        }
      }>
      <View
        accessibilityComponentType={undefined}
        accessibilityLabel={undefined}
        accessibilityTraits={undefined}
        accessible={true}
        hitSlop={undefined}
        isTVSelectable={true}
        onLayout={undefined}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "opacity": 1,
          }
        }
        testID={undefined}
        tvParallaxProperties={undefined}>
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Object {
              "backgroundColor": "#42033D",
              "borderColor": "black",
              "borderWidth": 2,
              "color": "#F7FFF7",
              "fontSize": 16,
              "height": 100,
              "margin": 20,
              "width": 160,
            }
          }>
          Discover
        </Text>
      </View>
      <View
        accessibilityComponentType={undefined}
        accessibilityLabel={undefined}
        accessibilityTraits={undefined}
        accessible={true}
        hitSlop={undefined}
        isTVSelectable={true}
        onLayout={undefined}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "opacity": 1,
          }
        }
        testID={undefined}
        tvParallaxProperties={undefined}>
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Object {
              "backgroundColor": "#42033D",
              "borderColor": "black",
              "borderWidth": 2,
              "color": "#F7FFF7",
              "fontSize": 16,
              "height": 100,
              "margin": 20,
              "width": 160,
            }
          }>
          Master Card
        </Text>
      </View>
    </View>
  </View>
</View>
`;
